app/assets/javascripts/tylium/modules/nodes.js.coffee

```diff
@@ -20,3 +20,12 @@ document.addEventListener "turbolinks:load", ->
   if $('body.nodes.show').length
     $('[data-behavior~=services-extras-link]').first().addClass('active')
     $('[data-behavior~=services-extras-tab-pane]').first().addClass('active')
+
+    $('[data-behavior~=evidence-dropdown-toggle]').click (e) ->
+      e.stopPropagation()
+      $('[data-name~=evidence]').click()
+
+    $('[data-behavior~=notes-dropdown-toggle]').click (e) ->
+      e.stopPropagation()
+      $('[data-name~=notes]').click()
+
```

Comment:
What's the deal here? We're taking a click from one place and pushing it onto something else because?

Reply:
This is so we can show the sidebar menu using the empty state CTA. We can't just add the same data attribute to the CTA because bootstrap expects the menu to be a sibling of the button